sql/2020/07_SEO/2019 Rewrites/http_status_codes.sql

```diff
@@ -0,0 +1,17 @@
+#standardSQL
+# 10_08: HTTP status codes returned
+SELECT
+  client,
+  status,
+  COUNT(0) AS freq,
+  SUM(COUNT(0)) OVER (PARTITION BY client) AS total,
+  ROUND(COUNT(0) * 100 / SUM(COUNT(0)) OVER (PARTITION BY client), 2) AS pct
+FROM
+  `httparchive.almanac.summary_requests`
+WHERE
+  firstReq
```

Comment:
Does this not need a date since we've now dated the almanac tables?

```suggestion
  `httparchive.almanac.summary_requests`
WHERE
  firstReq AND
  date = '2020-08-01'
```